cmd/generate/codeowners/codeowners.go

```diff
@@ -176,21 +181,24 @@ func run(opts *Options, cmd *cobra.Command) error {
 		return fmt.Errorf("error traversing git log: %w", err)
 	}
 
-	// Bootstrap codeowners
-	var outputPath string
+	// Add optional output path to base path
+	fullOutputPath := filepath.Join(opts.path, opts.outputPath)
```

Comment:
This creates some unexpected behavior and should be the fully resolved path of whatever the user's provided output path is. For example, if I give the following command:

```
❯ pizza generate codeowners . -l debug -o /tmp

Finished generating file: /Users/jpmcb/workspace/opensauced/pizza-cli/tmp/CODEOWNERS
```

I can see that it generated a file at `./tmp/CODEOWNERS` and not the root `/tmp` like I provided.

Reply:
I see –

From my understanding, I was assuming that the output directory would likely be wanted under the root of the current repository.

I'll work on generalizing this to align with what you pointed out. I'll aim to follow similar logic to that of the `-o` flag for `pizza generate config`

Reply:
> From my understanding, I was assuming that the output directory would likely be wanted under the root of the current repository.

That's one way to approach it. But, typically, when I've seen a `-o` output flag used, it's a fully qualified path. So, if you wanted to drop it in a `.bitbucket` directory, you'd just add the `./` to make it a fully resolvable path:

```sh
pizza generate codeowners . -o ./.bitbucket
```

but in my case, if I provide a full path (like `/tmp` or `~/.config/pizza-cli`), it should resolve to that path.

Reply:
@jpmcb – pushed a change; this functionality should now align with your expectations above.

Very similar logic to what @zeucapua used for the `-o` flag on `pizza generate config`

Do let me know if you'd like me to make this DRY and abstract it further, unless that's out of scope for this PR.